spec/support/chimpy.rb

```diff
@@ -3,4 +3,15 @@
   config.before do
     Spree::Chimpy.reset
   end
+
+  # rspec-rails 3 will no longer automatically infer an example group's spec type
+  # from the file location. You can explicitly opt-in to the feature using this
+  # config option.
+  # To explicitly tag specs without using automatic inference, set the `:type`
+  # metadata manually:
+  #
+  #     describe ThingsController, :type => :controller do
+  #       # Equivalent to being in spec/controllers
+  #     end
+  config.infer_spec_type_from_file_location!
```

Comment:
Is there a reason to prefer this setting (and explanation) over being explicit in the specs?